config/db.js

```diff
@@ -0,0 +1,20 @@
+require("dotenv").config();
+const mongoose = require("mongoose");
+const logger = require("../logger/createLogger");
+
+function connectDB() {
+    logger.info("Connecting...");
+    mongoose
+        .connect(process.env.MONGO_CONNECTION_URL, {
+            useNewUrlParser: true,
+            useUnifiedTopology: true,
+        })
+        .then(() => {
+            logger.info("MongoDB Connected");
+        })
+        .catch((err) => {
+            logger.info("Connection Failed ", err);
```

Comment:
Error handling needs to be improved, throw error from here

Reply:
where do i have to catch  these error then